### PR TITLE
info.json for mini1800

### PR DESCRIPTION
**Pull request by devbryce**

### CODE REVIEW

File: keyboards/lfkeyboards/mini1800/info.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "Mini1800", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "bootloader": "", 
+    "width": 19.5, 
+    "height": 5.25, 
+    "layouts": {
+        "DEFAULT": {
```
Comment (drashna) [inline keyboards/lfkeyboards/mini1800/info.json]: This needs to match the keyboard's "macro" name. 

In this case, it appears to be `LAYOUT`.
https://github.com/devbryce/qmk_firmware/blob/371bf146b5d454acb85eec1c6b5f9817c310d70a/keyboards/lfkeyboards/mini1800/mini1800.h#L72

File: keyboards/lfkeyboards/mini1800/info.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
...":"Enter", "x":12.75, "y":2, "w":2.25}, {"label":"4", "x":15.5, "y":2}, {"label":"5", "x":16.5, "y":2}, {"label":"6", "x":17.5, "y":2}, {"label":"+", "x":18.5, "y":2}, {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":1.75}, {"label":"1", "x":15.5, "y":3}, {"label":"2", "x":16.5, "y":3}, {"label":"3", "x":17.5, "y":3}, {"label":"Enter", "x":18.5, "y":3, "h":2}, {"label":"\u2191", "x":14.25, "y":3.25}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"Alt", "x":10, "y":4}, {"label":"Fn", "x":11, "y":4}, {"label":"Ctrl", "x":12, "y":4}, {"label":"0", "x":16.5, "y":4}, {"label":".", "x":17.5, "y":4}, {"label":"\u2190", "x":13.25, "y":4.25}, {"label":"\u2193", "x":14.25, "y":4.25}, {"label":"\u2192", "x":15.25, "y":4.25}]
```
Comment (noroadsleft) [inline keyboards/lfkeyboards/mini1800/info.json]: At the part where the split backspace is, take this:

`{"label":"Back Space", "x":13, "y":0}, {"label":"Del", "x":14, "y":0}, `

and change it to:

`{"label":"Back Space", "x":13, "y":0, "w":2}, `

**Comment by devbryce** [comment]
@drashna fixed the keycount/backspace issue.

**Comment by noroadsleft** [comment]
Line 9: `DEFAULT` still needs to be changed to `LAYOUT`.

**Comment by drashna** [comment]
Yup, but now you need to replace `DEFAULT` with `LAYOUT`, still.

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by drashna** [comment]
And it works!
https://config.qmk.fm/#/lfkeyboards/mini1800/LAYOUT

**Comment by devbryce** [comment]
Awesome, thanks again for your help the other night!

**Comment by drashna** [comment]
You're very welcome!

**Pull request merged**